### ISSUE TITLE
Fix credit_card_spec:24

### ISSUE THREAD

**Issue by michaelrkn** [epicodus/epicenter#103]
Waiting on reply from Balanced support.

**Comment by michaelrkn** [comment]
Also need to do the same for bank account - there's no pending spec there.

**Comment by michaelrkn** [comment]
Balanced does not support this. We could write a mock... but whatever, it's covered by an integration test.

**Comment by michaelrkn** [comment]
won't fix
